test/quantization/fx/test_model_report_fx.py

```diff
@@ -726,3 +703,100 @@ def forward(self, x):
         # run it through the model and do general tests
         for index, model in enumerate(models):
             self.run_model_and_common_checks(model, inputs[index], num_epochs, num_batches)
+
+
+"""
+Partition on domain / things to test
+
+There is only a single test case for now.
+
+This will be more thoroughly tested with the implementation of the full end to end tool coming soon.
+"""
+
+
+class TestModelReportDetectDynamicStatic(QuantizationTestCase):
```

Comment:
nit: please add `FX` to the name of the test case to indicate that this is related to fx

Reply:
is this needed when this class is already in a file that has fx in the name in directory called fx? noticed we have a lot of code like that

Reply:
Decided to go ahead and make this change so it is consistent with the naming convention right now (and can be changed to adhere to any naming convention changes later)